src/api/store/graph.py

```diff
@@ -87,6 +87,55 @@ def graph_actions_completed(self, context: Context, args) -> (Graph, MassEnergiz
       return None, CustomMassenergizeError(e)
 
 
+  def graph_actions_completed_by_team(self, context: Context, args) -> (Graph, MassEnergizeAPIError):
+    try:
+
+      team_id = args.get('team_id', None)
+
+      if not team_id:
+        return None, CustomMassenergizeError("Missing team_id field")
+
+      team: Team = Team.objects.get(id=team_id)
+
+      if not team:
+        return None, InvalidResourceError()
+
+      if context.is_prod and not context.user_is_admin():
+        if not team.is_published:
+          return None, CustomMassenergizeError("Content Not Available Yet")
+  
+      members = TeamMember.objects.filter(team=team).all()
+
+      completed_action_rels = []
+      for member in members:
+        completed_action_rels.extend(member.user.useractionrel_set.filter(status="DONE").all())
+
+      categories = TagCollection.objects.get(name="Category").tag_set.all()
+
+      prefetch_related_objects(completed_action_rels, "action__tags")
+      data = [
+        {
+          "id"   : category.id,
+          "name" : category.name,
+          "value": len(list(filter(lambda action_rel : category in action_rel.action.tags.all(), completed_action_rels)))
```

Comment:
i think you can take advantage of the sql generation to do a sort by.  I think django let's you fetch with a sort by

Reply:
done - see most recent commit